videostreamer.c

```diff
@@ -426,12 +430,14 @@ vs_write_packet(const struct VSInput * const input,
 				out_stream->time_base, AV_ROUND_NEAR_INF|AV_ROUND_PASS_MINMAX);
 	}
 
-	if (pkt->dts == AV_NOPTS_VALUE) {
-		pkt->dts = 0;
-	} else {
-		pkt->dts = av_rescale_q_rnd(pkt->dts, in_stream->time_base,
-				out_stream->time_base, AV_ROUND_NEAR_INF|AV_ROUND_PASS_MINMAX);
-	}
+	// TODO: is this duplicated? or is there a chance pkt->dts still
```

Comment:
I think we still need this. The `fix_dts` case above won't fire in all cases that `pkt->dts` could be `AV_NOPTS_VALUE`. Also, we would not be calling `av_rescale_q_rnd()` on it now.